website/blog/2022-05-27-aluminium.md

```diff
@@ -0,0 +1,280 @@
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+## Reduce file watcher memory usage
+Previously, Metals consumed a huge amount of memory for file watchers in large projects, especially on macOS.
+Now, Metals uses a memory-efficient way to watch files to detect changes and consumes less memory.
+
+For more technical details, see the original PR: [\#3758](https://github.com/scalameta/metals/pull/3758).
+
+## [Scala3] Override Completions
+Override completions for Scala3 is available with Metals 0.11.6!
+
+Now, it's possible to implement methods from the super class quite easily in Scala3.
+
+![override-completion](https://i.imgur.com/Go3sMxy.gif)
+
+## [Scala3] Show scaladoc on hover for Scala 3 project
+Now, Metals shows the scaladoc on hover for Scala3 project. (Before this release, Metals unable to show the scaladoc for the symbols from third-party modules).
+
+![hover-scala3](https://i.imgur.com/Svzq5DD.png)
+
+### [Scala3] Support `completionItem/resolve`
+Now, in Scala 3 we get scaladoc and shows default values when moving accross completions. 
+
+![completion-item-resolve](https://i.imgur.com/Tz6AOsx.gif)
+
+
+## Show parent scaladoc if implementation is returning empty
+
+Scaladocs can be inspected whenever you hover, use a completion or signature help. Up until recently we only showed you the documentation if the exact method you are using had the scaladocs written, which meant that if you overrode a method and didn't add the scaladoc comments again we would not show you any documentation. One example of such method could `headOption` on a `List`.
+
+ From this release we will also search the parent method in case the current method's scaladoc are empty. 
+
+## [Scala 3] Show scaladocs for signature help
+
+As mentioned in the previous paragraph, Metals can show you documentation in three different places. That, however, was true only for Scala 2 previously. In this release, we will now show you proper documentation whenever invoking signature help.
+
+As a remainder, signature help is used to indicate what parameters can be used in a method. It should pop up automatically after writing `(`, but you can also invoke it manually. In VS Code that takes the form of `editor.action.triggerParameterHints` command, which can also be bound to a shortcut and by default is.
+
+
+## [MUnit] Test Explorer can find helper methods from parent classes
+
+MUnit allows to use [helper functions](https://scalameta.org/munit/docs/tests.html#declare-tests-inside-a-helper-function) when declaring  tests. Very often those helper methods are extracted to some parent classes which are extended by many test suites.  
+Now, Metals can find usages of those helper methods and display them in Test Explorer.  
+
+This feature is available for Bloop and SBT 1.7.0-M2 or later.
```

Comment:
I think we can mention sbt 1.7.0-M2, as https://github.com/sbt/sbt/issues/6896 is a separate problem with this feature :) (anyway, I'll try to fix the problem in sbt).

Reply:
I'm afraid that some people will try `sbt 1.7.0-M2` and then come across https://github.com/sbt/sbt/issues/6896, and this could be very misleading experience that "Metals are not working". 
Alternatively, both `sbt 1.7.0-M2` and https://github.com/sbt/sbt/issues/6896 can be mentioned and then users will have a full picture.

Reply:
Ok then, we can drop sbt 1.7.0-M2 from release note :+1: (I don't have strong opinion).